webapp/packages/core-theming/src/ThemeSettingsManagementService.ts

```diff
@@ -16,14 +16,14 @@ export class ThemeSettingsManagementService extends Dependency {
   constructor(themeSettingsService: ThemeSettingsService, themeService: ThemeService, settingsManagerService: SettingsManagerService) {
     super();
     settingsManagerService.registerSettings(themeSettingsService.settings, () => [
-      {
-        key: 'defaultTheme',
-        type: ESettingsValueType.Select,
-        name: 'core_theming_settings_default_theme_name',
-        description: 'core_theming_settings_default_theme_description',
-        options: themeService.themes.map(theme => ({ id: theme.id, name: theme.name })),
-        group: INTERFACE_SETTINGS_GROUP,
-      },
+      // {
```

Comment:
I see no reason to comment other parts here
`webapp/packages/core-theming/src/ThemeSettingsService.ts`
or here
`webapp/packages/core-theming/src/ThemeService.ts`

Cause it just sets the theme to the light as default and does not affect anything

Reply:
exactly, we want to hide this setting from preferences only